js/zap-common.js

```diff
@@ -47,10 +62,17 @@ function parseYoutubeVideoID(url) {
     };
     var shortUrlDomain = "youtu.be";
     var longUrlDomain = "youtube.com";
+    var playlistUrlElement = "&list=";
```

Comment:
We need to be smarter about this, `list` could be the first query parameter - represented as `?list=foo`.
We should be able to check the return value of `getParameterByName(url, "list")` against being `null`, which means it's not in the URL.

Also, this means we don't have to declare this variable and it'll simplify the code below